### ISSUE TITLE
`TypeError` when calling `from_preset`

### ISSUE THREAD

**Issue by blair-ahlquist** [keras-team/keras-cv#1986]
I get a `TypeError` when calling `YOLOV8Detector.from_preset`:

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[4], line 1
----> 1 model = keras_cv.models.YOLOV8Detector.from_preset("resnet50_imagenet")
      3 println(model)

File [c:\Users\Blair\AppData\Local\Programs\Python\Python39\lib\site-packages\keras_cv\models\task.py:189](file:///C:/Users/Blair/AppData/Local/Programs/Python/Python39/lib/site-packages/keras_cv/models/task.py:189), in Task.__init_subclass__..from_preset(calling_cls, *args, **kwargs)
    188 def from_preset(calling_cls, *args, **kwargs):
--> 189     return super(cls, calling_cls).from_preset(*args, **kwargs)

File [c:\Users\Blair\AppData\Local\Programs\Python\Python39\lib\site-packages\keras_cv\models\task.py:136](file:///C:/Users/Blair/AppData/Local/Programs/Python/Python39/lib/site-packages/keras_cv/models/task.py:136), in Task.from_preset(cls, preset, load_weights, **kwargs)
    132     backbone_cls = keras.saving.get_registered_object(
    133         metadata["class_name"]
    134     )
    135     backbone = backbone_cls.from_preset(preset, load_weights)
--> 136     return cls(backbone, **kwargs)
    138 # Otherwise must be one of class presets
    139 config = metadata["config"]

TypeError: __init__() missing 2 required positional arguments: 'num_classes' and 'bounding_box_format'

**Comment by jbischof** [comment]
@blair-ahlquist you need to provide those two arguments along with the preset name to fully specify the model architecture. See our [OD guide](https://keras.io/guides/keras_cv/object_detection_keras_cv/) for more details.

**Comment by blair-ahlquist** [comment]
Ok. I was following the example listed [here](https://keras.io/api/keras_cv/models/tasks/yolo_v8_detector/).

**Comment by jbischof** [comment]
Good point, we should update the docstring! Feel free to submit a PR or we will get to it eventually.

**Comment by ianstenbit** [comment]
Thanks for finding this issue! If anyone would like to update this docstring, that would be great!
